online-rest-api/src/main/java/eu/itesla_project/online/rest/api/ApiOriginFilter.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2016, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package eu.itesla_project.online.rest.api;
+
+import java.io.IOException;
+
+import javax.servlet.*;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ *
+ * @author Giovanni Ferrari <giovanni.ferrari@techrain.it>
+ */
+public class ApiOriginFilter implements javax.servlet.Filter {
+
+    @Override
+    public void doFilter(ServletRequest request, ServletResponse response, FilterChain chain)
+            throws IOException, ServletException {
+
+        HttpServletResponse res = (HttpServletResponse) response;
```

Comment:
The doFilter method of the Filter is only called by the container when processing a request/response.
Do you want me to check for nulls anyway ?